fdbserver/workloads/AtomicOps.actor.cpp

```diff
@@ -268,23 +269,23 @@ struct AtomicOpsWorkload : TestWorkload {
 				memcpy(&intValue, kv.value.begin(), kv.value.size());
 				sum += intValue;
 				TraceEvent("AtomicOpOps")
-					.detail("Key", kv.key)
-					.detail("Val", kv.value)
-					.detail("IntVal", intValue)
-					.detail("CurSum", sum);
+				    .detail("Key", kv.key)
+				    .detail("Val", kv.value)
+				    .detail("IntVal", intValue)
+				    .detail("CurSum", sum);
 			}
-		} catch( Error &e ) {
+		} catch (Error& e) {
 			TraceEvent("DumpOpsKVError").detail("Error", e.what());
-			wait( tr.onError(e) );
+			wait(tr.onError(e));
 		}
 		return Void();
 	}
 
 	ACTOR Future<Void> validateOpsKey(Database cx, AtomicOpsWorkload* self, int g) {
+		state Key begin(format("debug%08x", g));
```

Comment:
This totally is a semantic change, woops. I'll fix it.

Reply:
@xumengpanda How much should I rewrite this? It's doing some strange things like doing point reads via getRange with a manually constructed single key range

Reply:
Huh, TIL strinc and keyAfter are different. My bad